src/components/Data/Indexes/Page.vue

```diff
@@ -126,42 +129,45 @@
                 class="mx-1"
                 title="browse this index"
                 variant="link"
-                :data-cy="`IndexesPage-browse--${row.item.indexName}`"
+                :data-cy="`IndexesPage-browse--${row.item.name}`"
                 :to="{
                   name: 'Collections',
-                  params: { index: row.item.indexName }
+                  params: { index: row.item.name }
                 }"
                 ><i class="fa fa-eye"></i
               ></b-button>
               <b-button
                 class="mx-1"
                 title="Create a collection in this index"
                 variant="link"
-                :data-cy="`IndexesPage-createCollection--${row.item.indexName}`"
+                :data-cy="`IndexesPage-createCollection--${row.item.name}`"
                 :to="{
                   name: 'CreateCollection',
-                  params: { index: row.item.indexName }
+                  params: { index: row.item.name }
                 }"
                 ><i class="fa fa-plus"></i
               ></b-button>
               <b-button
                 class="mx-1"
-                :data-cy="`IndexesPage-delete--${row.item.indexName}`"
+                :data-cy="`IndexesPage-delete--${row.item.name}`"
                 title="Delete index"
                 variant="link"
-                @click="openDeleteModal(row.item.indexName)"
+                @click="openDeleteModal(row.item)"
                 ><i class="fa fa-trash"></i
               ></b-button>
             </div>
           </template>
         </b-table>
       </template>
     </template>
-    <CreateIndexModal :id="createIndexModalId" />
+    <CreateIndexModal
+      :id="createIndexModalId"
+      @modal-close="onCreateModalClose"
+    />
     <DeleteIndexModal
       :id="deleteIndexModalId"
       :index="indexToDelete"
-      @modal-close="onModalClose"
+      @modal-close="onDeleteModalClose"
```

Comment:
BootstrapVue exposes a @hide event, right for this purpose, if I'm not mistaken.